awstransfer-s3-sam/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "Selective file transfer between SFTP server and S3 using AWS Transfer Family",
```

Comment:
Please keep the title at 75 characters or less

Reply:
Fixed.